content/topics/java-specific/introduction-to-spring-infrastructure/part-1/_index.md

```diff
@@ -2,6 +2,8 @@
 _db_id: 106
 content_type: topic
 title: Introduction to Spring Infrastructure part 1
+ready: true
```

Comment:
This content doesn't seem ready to me. Please only mark things as ready when they are in fact ready